mkdocs.yml

```diff
@@ -175,6 +175,7 @@ nav:
           - Speech Transcription: "tutorials/more_advanced/speech_transcription.ipynb"
           - Query Plan: "tutorials/more_advanced/query_plan.ipynb"
           - Named Entity Recognition: "tutorials/more_advanced/named_entity_recognition.ipynb"
+          - o1 Style Thinking: "tutorials/more_advanced/o1_style_thinking.ipynb"
```

Comment:
@KTS-o7 
Sorr, I forgot to mention one thing.

Reply:
Should i be doing anything 😅 Not so clear on what is expected of me now

Reply:
I'm sorry for my comment is not clear.
We had to add the notebook in the `nav` section in `mkdocs.yml`.
After that, we can see the notebook in the menu.
![image](https://github.com/user-attachments/assets/23f6beae-a47f-4dcb-86e6-41f8bcf443d1)

Reply:
Ok next time i will keep this in mind